deploy/backplane/cssre/config.yaml

```diff
@@ -5,4 +5,10 @@ selectorSyncSet:
       api.openshift.com/addon-rhmi-operator-internal: "true"
       api.openshift.com/addon-managed-api-service: "true"
       api.openshift.com/addon-managed-api-service-internal: "true"
+      deploymentMode: "SelectorSyncSet"
+      api.openshift.com/addon-kas-fleetshard-operator: "true"
+      api.openshift.com/addon-kas-fleetshard-operator-qe: "true"
+      api.openshift.com/addon-managed-kafka: "true"
+      api.openshift.com/addon-managed-kafka-qe: "true"
```

Comment:
Is this moving away from MKRTS? https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/mkrts/config.yaml#L4-L7

Reply:
Yes an no. We'll ultimately be responsible for the up time, and the new CSSRE members come from MKRTS, but I don't know that they'll want to remove their QE permissions ever even though production will go way after the transition.

Reply:
We don't make a differentiator between stage and prod. So this will need to go away completely if it's transitioning.

Reply:
That needs to be a conversation.  I can't just yank their permissions away right now, but my team does need permissions now.

Reply:
Removed the role.